### DISCUSSION TITLE
Target submenu item

### DISCUSSION THREAD

**Discussion by LiCempions** [moudey/Shell#475] [Q&A] [answered]
Hello.
I'm trying to group these items in two submenus, one for Microsoft (green) and one for Google (red), but I wasn't able to find a way to target items inside submenus using the modify section, for example:

`modify( find='Google*' image=#000000)`
I'm setting the image to a black square because I haven't created the two submenus yet.

![Screenshot Nilesoft Shell](https://github.com/moudey/Shell/assets/170435039/15fabbe9-89d3-44f8-9321-33c466294669)

I looked through the docs and google, so now I'm wondering if it is possible.

**Comment by iamLazyCode** [comment]
any luck iam trying to find solution for same

**Reply by LiCempions** [reply]
No I didn't find anything. Upvote if you want so we get some visibility...

**Reply by iamLazyCode** [reply]
> No I didn't find anything. Upvote if you want so we get some visibility...

done :)

**Comment by ghost** [comment, marked as answer]
First create the sub-menus and specify their parent menu:
```js
menu ( title='Microsoft' menu='Nuovo' ) {}
menu ( title='Google' menu='Nuovo' ) {}
```
Next move the desired entries to the created sub-menus:
```js
modify ( find='Microsoft|Rich' in='/Nuovo' menu='/Nuovo/Microsoft')
modify ( find='Google' in='/Nuovo' menu='/Nuovo/Google')
```
![image](https://github.com/moudey/Shell/assets/131535565/076b5cce-c20b-47e5-8271-855d2e5a53ca)

**Reply by iamLazyCode** [reply]
> First create the sub-menus and specify their parent menu:
> 
> ```js
> menu ( title='Microsoft' menu='Nuovo' ) {}
> menu ( title='Google' menu='Nuovo' ) {}
> ```
> 
> Next move the desired entries to the created sub-menus:
> 
> ```js
> modify ( find='Microsoft|Rich' in='/Nuovo' menu='/Nuovo/Microsoft')
> modify ( find='Google' in='/Nuovo' menu='/Nuovo/Google')
> ```
> 
> ![image](https://private-user-images.githubusercontent.com/131535565/346591134-076b5cce-c20b-47e5-8271-855d2e5a53ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0NTY5MDksIm5iZiI6MTcyMDQ1NjYwOSwicGF0aCI6Ii8xMzE1MzU1NjUvMzQ2NTkxMTM0LTA3NmI1Y2NlLWMyMGItNDdlNS04MjcxLTg1NWQyZTVhNTNjYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOFQxNjM2NDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZDliZDI3YzhjZGUyMDI2NjU2ZmU5Nzc5N2U3NGE5YTRhZTUwNzIwYWM5ZWIyYWE3NGYxY2MyZjVkMWFjZTIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YBAplwB1WwZScsfdQXVaxNn8j-M6XyilmvKY-qqURgs)

absolutely great work sir
